### ISSUE TITLE
Mortal Kombat 4

### ISSUE THREAD

**Issue by epayson85** [loganmc10/GLupeN64#59]
In android it works perfectly except it doesn't show the health bar / time at the top of the screen

on windows it crashes after you select a player and go to choose which difficulty you want to play

**Comment by orbea** [comment]
I can reproduce this, however even with `DEBUG=1` I am getting an incredibly non-useful backtrace...

**Comment by loganmc10** [comment]
This happens even in Linux? I'll check it out tomorrow, I was distracted by this new world of emscripten...

**Comment by orbea** [comment]
Yea, it happens in Slackware for me. The lack of debugging symbols even when I should have them is pretty strange...

**Comment by loganmc10** [comment]
I can reproduce it as well, I've never seen a backtrace like that before, especially with DEBUG on...

```
(gdb) backtrace
#0  0x0000000000000000 in ?? ()
#1  0x3f52d2d43f52d2d4 in ?? ()
#2  0x3f0080813f52d2d4 in ?? ()
#3  0x0000000000000000 in ?? ()
```

**Comment by loganmc10** [comment]
It has something to do with the OpenGL VBO code (GLupeN64 uses a modified version of GLideN64 that implements VBO's, from here: https://github.com/loganmc10/GLideN64/tree/vbo)

Disabling VBO's fixes the crash.

**Comment by loganmc10** [comment]
I filed an upstream issue:
https://github.com/gonetz/GLideN64/issues/1130

For some reason the behaviour in Mortal Kombat doesn't cause a segfault when not using VBO's, but it is still bad behaviour that is just kind of "showcased" when using VBO's.

**Comment by loganmc10** [comment]
This should be fixed in Windows now, but won't be fixed on Linux until there is some kind of fix in GLideN64

**Comment by loganmc10** [comment]
Fixed by 87326cede6a4d8730ec608bb3fc928402490d3d6

**Comment by orbea** [comment]
This is also fixed for me, thanks.
